ProcessLib/ComponentTransport/CreateComponentTransportProcess.cpp

```diff
@@ -105,7 +107,7 @@ std::unique_ptr<Process> createComponentTransportProcess(
 
     // Collect all process variables in a vector before allocation
     // pressure first, concentration then
-    auto const collected_process_variables = findProcessVariables(
+    auto collected_process_variables = findProcessVariables(
```

Comment:
minor: I didn't get it why it is not longer const.
I'd even make it a `const &` if possible. (Avoids a copy of a vector, though the gains are minuscule, but semantics is much better.)

Reply:
my bad. fixed.